ydb/core/kqp/session_actor/kqp_query_state.cpp

```diff
@@ -143,16 +150,19 @@ std::unique_ptr<TEvKqp::TEvCompileRequest> TKqpQueryState::BuildCompileRequest(s
         case NKikimrKqp::QUERY_ACTION_PREPARE:
             query = TKqpQueryId(Cluster, Database, GetQuery(), settings, GetQueryParameterTypes());
             keepInCache = query->IsSql();
+            perStatementResult = false;
```

Comment:
So there will be a difference between two ways of sql execution:
1. ExecuteQuery with QUERY_ACTION_EXECUTE
2. ExecuteQuery with QUERY_ACTION_PREPARE and then QUERY_ACTION_EXECUTE_PREPARED

I understand that we can't execute DDL and DML in one SQL without compiling queries after execution of previous statements (fresh tables are in DMLs). But do we realize what exactly will be different in usual DML queries?